src/docs/product/error-monitoring/issue-details/index.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Issue Details
+sidebar_order:
+description: "Learn more about Issue Details page."
+---
+
+The Issue Details page provides you with information to help you to get further insight into the reasons for the issue. All the informtion about the issue that's on the Issues page is displayed on the Issue Details page as well. This page displays information about a specific error event that's part of an issue, however, you can navigate between the events of an issue using the "Older" and "Newer" buttons.
+
+TO DO Add screenshot here of Issue Details page
+
+To access this page, click the any issue on the Issues page.
+
+This page opens on the Details tab, but there are several other tabs as well:
+
+- Activity: A history of activity on the for the event where team members can post comments.
+- User Feedback: Any feedback collected from users through Sentry's User Feedback widget. TO DO Add link here to take them to Platform selection page but that gets them to User Feedback
```

Comment:
FYI - we have both the User Feedback widget and the API, so suggest rephrasing as edited here. Also added the platform redirect link (which uses the context of the reader's platform, if known, to direct them to the correct content. If not known, the platform selector intervenes, they select the platform and are then directed to the User Feedback content):
```suggestion
- User Feedback: Any feedback collected from users through the <PlatformLink to="/enriching-events/user-feedback/">SDK's user feedback</PlatformLink>. 
```